.changelog/1268.bugfix.md

```diff
@@ -0,0 +1 @@
+Don't die on missing runtime TX to address
```

Comment:
this and PR description are confusing

Reply:
how about now?

Reply:
How about still confusing?

`Fix optional field to rendering in RuntimeTransactions`

> For some reason we assumed that there is always a tx.to_eth or tx.to field, but that assumption is true, so don't make it.

It was previously presumed that a ‘tx.to_eth’ or ‘tx.to’ field is present, However, both fields are optional.

Reply:
Updated accordingly.